### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by wolterbwambale**
### In this project, i have implemented the following:-

- [ ] Add styling 
- [ ] Add JSX elements to your project without functionality
- [ ]  i have follow Zeplin to style the website

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,4 +2,205 @@ body {
...
+  /* justify-content: space-between; */
+  background-color: #fff;
+  padding: 10px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: bold;
+  color: rgb(91, 91, 247);
```
Comment (joskalenda) [inline src/App.css]: - [ ] the color you are using for your design is not what used in the original template, `#0290ff`, please make sure you update this

File: src/components/Navbar.js [hunk 2]
```diff
@@ -3,13 +3,13 @@ import { Link } from 'react-router-dom';
 
 function Navbar() {
   return (
-    <nav>
-      <div>Bookstore CMS</div>
+    <nav className="navbar">
+      <div className="logo">Bookstore CMS</div>
       <ul>
```
Comment (joskalenda) [inline src/components/Navbar.js:8]: - [ ] Check this out,


1. Your book is missing some alignment on the top, bottom, and left as shown and compared to the original template
2. The bg-color of your design is `gray` while on the original design, it is `white`
3. Your book is missing a small border-radius property as shown and compared to the original design

File: src/components/categories.js [hunk 3]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 function Categories() {
   return (
-    <div><h1>This is categories sections</h1></div>
+    <div>
+      <h1>This is categories sections</h1>
+    </div>
   );
 }
```
Comment (joskalenda) [inline src/components/categories.js:7]: - [ ] This link to your [github profile](https://github.com/wolterbwambale/bookstore) in your readme file is broken, please update it

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @wolterbwambale , |
|Good job so far! You still need to work on some issues to go to the next project, but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Readme file `97%...`
- [ ].. ✅ Great design `98%...`
- [ ].... ✅ Clean code `cheers...`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @wolterbwambale       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive pr👍 
- Good implementation of the required changes 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**